backend/app/tests/crud/test_patient.py

```diff
@@ -0,0 +1,59 @@
+
+import uuid
+from app.models import PatientCreate, PatientUpdate, UserCreate
+from app.tests.utils.utils import random_email, random_lower_string, random_phone_number
+from sqlmodel import Session
+from datetime import datetime
+from app import crud
+
+
+
+
+def test_patient(db: Session) -> None:
```

Comment:
This is a good and working solution, but we're here to learn what the cool girls do.
Pytest fixtures may help you. Try and see if a "caregiver_Fixture" is a viable option here.

https://stackoverflow.com/questions/33508060/create-and-import-helper-functions-in-tests-without-creating-packages-in-test-di